src/run-mats

```diff
@@ -20,14 +20,20 @@ case $# in
   *) fail;;
 esac
 
-if [ ! -d "${LIBDIR}" ]; then
-  echo "${this} expected to find $LIBDIR"
-  echo ""
+GITTOP="$(git rev-parse --show-toplevel)"
+case "$(uname -s)" in
+  MINGW64_NT-* | MSYS_NT-*)
+    GITTOP="$(cygpath -u "$GITTOP")"
```

Comment:
Interesting. I take it `git` is returning a Windows path on those systems?

Reply:
It is, which was quite surprising.